tools/downloader/converter.py

```diff
@@ -107,8 +108,15 @@ def main():
         try:
             mo_path = Path(os.environ['INTEL_OPENVINO_DIR']) / 'deployment_tools/model_optimizer/mo.py'
         except KeyError:
-            sys.exit('Unable to locate Model Optimizer. '
-                + 'Use --mo or run setupvars.sh/setupvars.bat from the OpenVINO toolkit.')
+            try:
+                # For OpenVINO from PYTHONPATH
+                mo_path = subprocess.run([args.python, '-c',
+                                          'import mo,sys; sys.stdout.write(mo.__file__)'],
+                                         check=True, stdout=subprocess.PIPE, encoding='utf-8').stdout
+                mo_path = (Path(mo_path).parent / '__main__.py').resolve(strict=True)
+            except Exception:
```

Comment:
```suggestion
            except (subprocess.CalledProcessError, OSError):
```

Any other exception likely means that the code is broken, not that MO is not found.